stackslib/src/chainstate/nakamoto/miner.rs

```diff
@@ -176,13 +179,19 @@ impl NakamotoBlockBuilder {
     ///
     /// * `coinbase` - the coinbase tx if this is going to start a new tenure
     ///
+    /// * `bitvec_len` - the length of the bitvec of reward addresses that should be punished or not in this block.
+    ///
+    /// * `cost_limit_percentage` - the percentage of the remaining tenure cost limit to use when
```

Comment:
This now has a mismatch of the actual function arg

Reply:
Ah thanks. Will fix.